build/build.xml

```diff
@@ -1,4 +1,4 @@
 <build>
-  <project id="Examine" version="1.0.6"/>
-  <project id="Examine.AzureDirectory" version="1.0.0-beta07"/> 
+  <project id="Examine" version="1.0.7"/>
+  <project id="Examine.AzureDirectory" version="2.0.1"/> 
```

Comment:
That directory never left beta state it means even if we are introducing breaking changes we dont need bump versions.